prow/jenkins/controller.go

```diff
@@ -90,30 +104,7 @@ func (c *Controller) incrementNumPendingJobs(job string) {
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
```

Comment:
Nit: we should probably make the blank line here consistent between getNumPendingJobs and incrementNumPendingJobs (remove it?)

Reply:
Yeah, I just did it w/o reading this comment :)